policyengine_us/parameters/gov/states/mn/tax/income/amt/fractional_income_threshold.yaml

```diff
@@ -5,24 +5,40 @@ metadata:
   unit: currency-USD
   breakdown:
   - filing_status
+  uprating:
+    parameter: gov.irs.uprating
+    rounding:
+      type: nearest
+      interval: 10
   reference:
+  - title: 2023 Minnesota Statutes Section 290.091 — Alternative Minimum Tax On Preference Items Subd.3.a
```

Comment:
```suggestion
  - title Minnesota Statutes Section 290.091 — Alternative Minimum Tax On Preference Items Subd.3.a
```

Reply:
2023 in the legal code would imply that there is a new link every year but I believe the same link will be updated